ql-qlf-plugin/ql-qlf-k6n10-plugin/synth_quicklogic.cc

```diff
@@ -0,0 +1,280 @@
+/*
```

Comment:
Do not add synth_quicklogic.cc separately for different devices. Instead add only synth_quicklogic.cc which support both the devices.

Reply:
> @lnsharma @kgugala, i see some confusion around proposal for plugin. I think lalit is suggesting to create 2 different plugin (one .so per family), however he is proposing to keep synth_quicklogic.cc common. Whereas what Karol is proposing that there should be single plugin (a single .so) handling both families. Is that correct understanding I have?

If we have one synth_quicklogic.cc, then it is obvious that there is going to be one .so. Correct me if I am wrong. That is what I have been saying to have one synth_quicklogic.cc.

Reply:
> > @lnsharma @kgugala, i see some confusion around proposal for plugin. I think lalit is suggesting to create 2 different plugin (one .so per family), however he is proposing to keep synth_quicklogic.cc common. Whereas what Karol is proposing that there should be single plugin (a single .so) handling both families. Is that correct understanding I have?
> 
> If we have one synth_quicklogic.cc, then it is obvious that there is going to be one .so. Correct me if I am wrong. That is what I have been saying to have one synth_quicklogic.cc.

Ok. Thanks for the clarifying this.

Reply:
I've merged the 2 synth_quicklogic.cc, we now have one plugin for both family.